### DISCUSSION TITLE
代理winscp出现断流问题

### DISCUSSION THREAD

**Discussion by qiuzi** [eycorsican/leaf#74] [Q&A]
使用透明代理winscp 每2分钟会出现连接断开问题
使用的是wss+cdn
ws error: WebSocket protocol error: Connection reset without closing handshake [Trojan]

**Comment by eycorsican** [comment]
用的 cloudflare 吧？

**Reply by qiuzi** [reply]
是的 如果增加smux特性会大大改善

**Comment by eycorsican** [comment]
Cf 闲置100秒超时，keep alive可解

**Reply by DuckSoft** [reply]
可能随机空置的时候发个 Ping Frame 就好了？
还得看远端实现情况……

**Reply by qiuzi** [reply]
是在leaf里配置keep alive吗？

**Reply by ghost** [reply]
> Cf 闲置100秒超时，keep alive可解
> 
> 

leaf设置吗
